deploy/scripts/src/accounts.ts

```diff
@@ -212,21 +212,21 @@ export function setup (): void {
 }
 
 const env = {
-  envURL: `${__ENV.HOME_URL}`,
-  signinURL: `${__ENV.SIGNIN_URL}`
+  envURL: `${__ENV.ACCOUNTS_HOME_URL}`,
+  signinURL: `${__ENV.ACCOUNTS_SIGNIN_URL}`
```

Comment:
```suggestion
  envURL: __ENV.ACCOUNTS_HOME_URL,
  signinURL: __ENV.ACCOUNTS_SIGNIN_URL
```
This can be fixed too to make it more consistent whilst we are changing this line